site/source/tutorials/1020_registries/index.md

```diff
@@ -96,31 +96,27 @@ First we need to extract the `_renderBack` function from `Worker.ts` into a new
 
 {% task 'Add `WorkerBack` import to `Worker.ts`' %}
 
-{% include_codefile 'demo/finished/biz-e-corp/src/widgets/Worker.ts' line:5 %}
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/Worker.ts' line:7 %}
 
 {% task 'Update the `_renderBack` function to use the `WorkerBack` registry items in `Worker.ts`' %}
 
-{% include_codefile 'demo/finished/biz-e-corp/src/widgets/Worker.ts' lines:51-65 %}
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/Worker.ts' lines:56-70 %}
 
 {% aside 'Use Before You Define' %}
 The `registry` is designed to mirror the behavior and API of custom elements wherever possible. One neat feature is that a registry item can be used before it is defined, and once defined, widgets that use the `registry` will automatically re-render!
 {% endaside %}
 
-Now we need to add the `registry` definition for `WorkerBack.ts` to lazily load when the worker is clicked. Instead of adding a concrete widget class, we add a function that, when called, requires and returns the concrete widget. This function will not be called until the first time the application tries to use the widget label as part of the usual `render` cycle. Initially, before the widget has loaded, nothing will be rendered. Once it has loaded, any widgets that use the lazy widget will automatically re-render.
+Now we need to add the `registry` definition for `WorkerBack.ts` to lazily load when the worker is clicked. Instead of adding a concrete widget class, we add a function that, when called, dynamically imports the widget and returns the a promise that returns the widget. This function will not be called until the first time the application tries to use the widget label as part of the usual `render` cycle. Initially, before the widget has loaded, nothing will be rendered. Once it has loaded, any widgets that use the lazy widget will automatically re-render.
```

Comment:
returns the a promise that returns the widget ->  returns a promise that returns the widget